geomesa-convert/geomesa-convert-common/src/main/resources/META-INF/services/org.geotools.filter.expression.PropertyAccessorFactory

```diff
@@ -1 +1 @@
-org.locationtech.geomesa.convert.cql.ArrayPropertyAccessorFactory
+org.locationtech.geomesa.convert2.transforms.CqlFunctionFactory$ArrayPropertyAccessorFactory
```

Comment:
Are users going to have problems finding the right jars to include?  (I.e., is there a new convert2 jar that folks will need to be educated about?)

Reply:
no, both packages are in the same geomesa-convert jar (java 9 modules? what?)